src/components/ItemStatus/ItemStatus.container.ts

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux'
+
+import { RootState } from 'modules/common/types'
+import { MapStateProps, MapDispatchProps, MapDispatch, OwnProps } from './ItemStatus.types'
+import { getStatusByItemId } from 'modules/item/selectors'
+import ItemStatus from './ItemStatus'
+
+const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
+  const statusByItemId = getStatusByItemId(state)
+  return {
+    status: statusByItemId[ownProps.item.id]
+  }
+}
+
+const mapDispatch = (_dispatch: MapDispatch): MapDispatchProps => ({})
+
+export default connect(mapState, mapDispatch)(ItemStatus)
```

Comment:
Instead of passing an empty mapDispatch you can always pass null or nothing at all.
```
export default connect(mapState)(ItemStatus)
```

Reply:
Yea to be honest I just leave them like that to make it easier to connect later on which is usually what happens with most components over time :P (that's why i create the types too even tho they are empty)